package.json

```diff
@@ -5,32 +5,33 @@
   "main": "index.js",
   "repository": {
     "type": "git",
-    "url": "git@bitbucket.org:rulemotion/resin-jwt"
+    "url": "git@github.com:resin-io-modules/resin-jwt"
   },
   "author": "Aleksis Brezas <abresas@resin.io>",
   "scripts": {
     "lint": "resin-lint ./index.coffee ./test",
     "pretest": "npm run lint && coffee -c index.coffee",
-    "test": "mocha --compilers coffee:coffee-script/register test/index.coffee",
+    "test": "mocha --require coffeescript/register test/index.coffee",
     "prepublish": "npm test"
   },
   "dependencies": {
     "bluebird": "^3.0.0",
-    "jsonwebtoken": "^5.0.4",
-    "passport-jwt": "^2.1.0",
+    "jsonwebtoken": "^8.3.0",
+    "passport-jwt": "^4.0.0",
     "request": "^2.65.0"
   },
   "devDependencies": {
     "atob": "^2.0.0",
-    "chai": "^3.0.0",
-    "chai-as-promised": "^5.1.0",
-    "coffee-script": "~1.10.0",
+    "chai": "^4.2.0",
+    "chai-as-promised": "^7.1.1",
+    "coffeescript": "^2.3.2",
```

Comment:
This is potentially a breaking change as I know coffeescript 2 changes the target es version for compilation compared to coffeescript 1

Reply:
fixed